CHANGELOG.md

```diff
@@ -52,6 +52,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [2.0.2] - 2023-06-28
 
+### Added
```

Comment:
@sanAkel Can you move your changelog update up to the `[Unreleased]` section of the changelog?

Reply:
Its done @mathomp4